WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -416,6 +426,40 @@ public boolean onActivityBackPressed() {
         return goBackInPostHistory();
     }
 
+    @Override
+    public void onFollowTapped(View view, String blogName, final long blogId) {
+        mDispatcher.dispatch(AccountActionBuilder.newFetchSubscriptionsAction());
+
+        Snackbar.make(view, Html.fromHtml(getString(R.string.reader_followed_blog_notifications,
+                        "<b>", blogName, "</b>")), Snackbar.LENGTH_LONG)
```

Comment:
There are some cases when blog name is empty. Should we modify copy to reflect this or leave it like this?

[![https://gyazo.com/b4a23410102059d6d12b55f26a6c2d4b](https://i.gyazo.com/b4a23410102059d6d12b55f26a6c2d4b.png)](https://gyazo.com/b4a23410102059d6d12b55f26a6c2d4b)

Reply:
I updated the copy to say "Enable notifications for this site?" for that case in 65c6da9.  What do you think, @SylvesterWilmott?